diff --git a/config.c b/config.c
index ac7358f..57ade17 100644
--- a/config.c
+++ b/config.c
@@ -4,7 +4,7 @@

 #include <ctype.h>
 #ifndef _MSC_VER
-#include <iniparser.h>
+#include <iniparser/iniparser.h>
 #endif
 #include <math.h>

diff --git a/configure.ac b/configure.ac
index 5aeda60..bac4f72 100644
--- a/configure.ac
+++ b/configure.ac
@@ -364,14 +364,8 @@ AS_IF([test "x$enable_output_ncurses" != "xno"], [
   )

   AC_CHECK_LIB($curses_lib, initscr,
-    if test "$CURSES_CONFIG" = "" ; then
-      LIBS="$LIBS -l$curses_lib"
-      CPPFLAGS="$CPPFLAGS -DNCURSES"
-    fi
-    if test "$CURSES_CONFIG" != "" ; then
-      CPPFLAGS="$CPPFLAGS `$CURSES_CONFIG --cflags` -DNCURSES"
-      LIBS="$LIBS `$CURSES_CONFIG --libs`"
-    fi
+    LIBS="$LIBS -l$curses_lib"
+    CPPFLAGS="$CPPFLAGS -DNCURSES"

     AC_CHECK_HEADERS([curses.h], , AC_MSG_ERROR([missing curses.h header]))
     have_ncurses=yes,
@@ -426,7 +420,7 @@ AC_CHECK_LIB(iniparser,iniparser_load, have_iniparser=yes, have_iniparser=no)
     if [[ $have_iniparser = "yes" ]] ; then
     LIBS="$LIBS -liniparser"
       CPPFLAGS="$CPPFLAGS -I/usr/include/iniparser"
-    AC_LINK_IFELSE([AC_LANG_PROGRAM([[#include <iniparser.h>]],
+    AC_LINK_IFELSE([AC_LANG_PROGRAM([[#include <iniparser/iniparser.h>]],
       [[dictionary* ini;
       const char *keys[3];
       iniparser_getseckeys(ini, "eq", keys);]])],

